### PR TITLE
Update `Global Moderation Team`, `Nomination Assessment Team`, and `osu! Alumni` for February 2024

### PR DESCRIPTION
**Pull request by aceticke**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

Postponing BN for when I have more time, since there are considerably more changes to do for those pages.

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/en.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | French, Spanish | Chat moderation |
 | ::{ flag=ID }:: [Shurelia](https://osu.ppy.sh/users/3807986) | Indonesian | Chat moderation |
 | ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) | Indonesian, some Vietnamese | Chat moderation, forum moderation |
+| ::{ flag=AT }:: [Stefan](https://osu.ppy.sh/users/626907) | German | Forum moderation |
```
Comment (iSlodinxOsu) [inline wiki/People/Global_Moderation_Team/en.md:138]: I think he also speaks Serbian iirc

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/People/Global_Moderation_Team/en.md (hunk 1 above)
Comment (iSlodinxOsu) [reply wiki/People/Global_Moderation_Team/en.md:138]: Let me ask him real quick if it's true.

Comment (iSlodinxOsu) [reply wiki/People/Global_Moderation_Team/en.md:138]: Oh okay now i remember when i asked him once before, he's actually Serbian.

Comment (aceticke) [reply wiki/People/Global_Moderation_Team/en.md:138]: Internally he isn't listed for that language, just OnosakiHito and 0x84f come to mind. Don't think he's moderating it?

Comment (iSlodinxOsu) [reply wiki/People/Global_Moderation_Team/en.md:138]: Not sure, but all i know that he is also Serbian, didn't ask if he also speaks Serbian, probably it should be obvious? I don't really know...

Comment (aceticke) [reply wiki/People/Global_Moderation_Team/en.md]: He hath become Serbian

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**